app/controllers/black_lists_controller.rb

```diff
@@ -0,0 +1,24 @@
+#  Copyright (c) 2018, Pfadibewegung Schweiz. This file is part of
+#  hitobito_pbs and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_pbs.
+
+class BlackListsController < SimpleCrudController
+
+  self.permitted_attrs = [:first_name, :last_name, :pbs_number, :email,
+                          :phone_number, :reference_name, :reference_phone_number]
+
+  def list_entries
+    super.page(params[:page])
+  end
+
+  private
+
+  def black_list
+    @black_list ||= BlackList.find(params[:id])
```

Comment:
Oh, I never really made some thoughts about this. It was scaffolded at the first step and never touched again. I will remove it because it will also work with `entry`